crates/icrate/Cargo.toml

```diff
@@ -163,6 +169,26 @@ unstable-example-browser = [
     "WebKit_WKNavigation",
     "WebKit_WKWebView",
 ]
+unstable-example-metal = [
```

Comment:
If you add this to `header-translator/src/output.rs` under e.g. `macos_11_features` (or whatever version is required), then the CI will be check the example